src/components/structures/TimelinePanel.tsx

```diff
@@ -1115,19 +1115,24 @@ class TimelinePanel extends React.Component<IProps, IState> {
     public forgetReadMarker = (): void => {
```

Comment:
Does this PR also fix https://github.com/vector-im/element-web/issues/1681?

Any change to https://github.com/vector-im/element-web/issues/10873?